tailwind.config.js

```diff
@@ -2,6 +2,23 @@ const defaultTheme = require('tailwindcss/defaultTheme')
 
 module.exports = {
   theme: {
+    colors: {
+      white: '#ffffff',
```

Comment:
Tailwind ships with a white color by default, right? Not sure why you added this?

Reply:
As I understand it, adding colors to the `theme` section will override all of the tailwind colors and so white needed recreating.

I can add the colors to the `extend` section instead. However, it's worth bearing in mind that if we define our own colors, the compiled css file size will be smaller as we are not including colours we don't need.

Reply:
You can place the colors under the extend to override a single color: https://tailwindcss.com/docs/customizing-colors/#overriding-a-default-color

Is the filesize that much smaller without the other colors? I could get behind that idea but if it's not that much then I don't know if it's worth fiddling with the defaults that much.

Reply:
After running `npm run production`, the output file sizes are as follows:

Custom colours: 189KB
Default colours: 363KB

This doesn't take gzipping into account.

Reply:
Great. Definitely worth it then 👍

Reply:
OK cool 😎

Based on the two conversation threads related to this file, I'm going to go for something like:

```
module.exports = {
  theme: {
    colors: {
      white: '#ffffff',
      green: {
        primary: '#18bc9c',
        dark: '#15a589',
        darker: '#12826c'
      },
      gray: {
        ...defaultTheme.colors.gray
      }
    },
    extend: {
      fontFamily: {
        sans: [
          'Lato',
          ...defaultTheme.fontFamily.sans,
        ]
      },
    }
  },
  variants: {},
  plugins: []
}
```

Does that look ok to you?

Reply:
Yep. Perfect!